content/shared/v3-core-get-started/_index.md

```diff
@@ -327,7 +327,28 @@ For more information, see [diskless architecture](#diskless-architecture).
 > Because InfluxDB sends a write response after the WAL file has been flushed to the configured object store (default is every second), individual write requests might not complete quickly, but you can make many concurrent requests to achieve higher total throughput.
 > Future enhancements will include an API parameter that lets requests return without waiting for the WAL flush.
 
-#### Create a database or table
+##### No sync write option
+
+InfluxDB provides a `no_sync` write option to allow faster response of write request by skipping the wait for WAL presistence. When `no_sync=true`, InfluxDB writes data to WAL but immediately acknowledges the write request without waiting for persistence. 
+
+- Default behavior(`no_sync=false`): Ensures data is persisted before acknowledging writes. Thus, reducing the risk of data loss. 
+- With `no_sync=true`: Improves write performance but increases the risk of data loss in case of crashes before WAL persistence. 
```

Comment:
```suggestion
- With `no_sync=true`: Reduces write latency but increases the risk of data loss in case of crashes before WAL persistence. 
```

Reply:
Use extreme caution about making any claims for or against "performance"--the word carries a lot of weight.